File: resourcespoet/src/test/java/com/commit451/resourcespoet/IdTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

/**
 * Tests the resources creation
 */
public class IdTest {

    @Test
    public void idTest() throws Exception {
        ResourcesPoet poet = ResourcesPoet.create()
                .addId("some_id");

        TestUtil.assertEquals("id.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/IdTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class IdTest {

    @Test
    fun idTest() {
        val poet = ResourcesPoet.create()
            .addId("some_id")

        TestUtil.assertEquals("id.xml", poet)
    }
}
